language/move-lang/src/parser/syntax.rs

```diff
@@ -1870,6 +1871,15 @@ fn parse_condition<'input>(tokens: &mut Lexer<'input>) -> Result<SpecBlockMember
             parse_exp,
             "an aborts code or modifies target",
         )?
+    } else if kind == SpecConditionKind::Emits {
+        consume_identifier(tokens, "to")?;
+        let mut additional_exps = vec![parse_exp(tokens)?];
+        if tokens.peek() == Tok::If {
```

Comment:
You can use match_token here instead of peek/advance

Reply:
Done. Thanks!